### PR TITLE
Event fire second parameter

### PR DESCRIPTION
**Pull request by dalderliesten**
-Updated the parameters of the fire event to reflect the new, second parameter.
-Updated the scene display of the fire event to include this parameter on the display.

Please only merge this branch once approved and in conjunction with the PR for the client.

ONLY MERGE IN CONJUNCTION WITH CLIENT PR 51.

### CODE REVIEW

**Comment by jessetilro** [comment]
Code looks very good. Will manually test in 45 minutes. You mention a PR in the description, please include a link to it.

**Comment by dalderliesten** [comment]
Haha, I also added a DO NOT MERGE, since that PR is still being tested.  Will notify you.

**Comment by wouterz** [comment]
Fire event works, completion audio is broken. I suggest removing this

File: src/main/java/nl/tudelft/pixelperfect/event/type/FireOutbreakEvent.java [hunk 1]
```diff
@@ -13,53 +13,48 @@
...
-
-      setNotifiedFlag(true);
-    }
-  }
+	/**
+	 * Constructor for FireOutbreakEvent event.
+	 *
+	 * @param id
+	 *            The desired id.
+	 * @param summary
```
Comment (wouterz) [inline src/main/java/nl/tudelft/pixelperfect/event/type/FireOutbreakEvent.java]: Check your indentation for this complete file

**Comment by jessetilro** [comment]
Your latest commit messes up the indentation again. Please check your Eclipse settings.

**Comment by wouterz** [comment]
Why not completely remove it, leaving a lot of empty methods?

**Comment by dalderliesten** [comment]
Jesse wishes to use it later, after this deadline.  Also useful for any final things we wish to add, as the method itself works fine.

**Comment by wouterz** [comment]
Okay, in then im merging both PR's after this travis build

**Pull request merged**